### DISCUSSION TITLE
[iOS] text alignment wrong when set line height

### DISCUSSION THREAD

**Discussion by wgywgy** [reactwg/react-native-new-architecture#83] [Q&A] [answered]
When I run rn-tester project, the text alignment wrong when set line height.
RN Version 0.68.1
Here is the code:
```
{
    title: 'Line Height',
    render: function (): React.Node {
      return (
        <Text>
          <Text style={{lineHeight: 35, backgroundColor: '#eeeeee'}}>
                      A lot of spssssace between the lines of this long passage that should
                      wrap once.
          </Text>
        </Text>
      );
    },
  },
```

And Display the text align vertical bottom:
![Simulator Screen Shot - iPhone 13 Pro - 2022-10-10 at 14 29 50](https://user-images.githubusercontent.com/2408246/194808998-ea48d9d5-8a00-491a-a3f0-9da4141d4ddc.png)

**Comment by cipolleschi** [comment]
Hi @wgywgy! Thanks for reporting this, we are working on a fix! 👍

**Comment by sammy-SC** [comment, marked as answer]
Hello @wgywgy,

thank you for the report.

https://github.com/facebook/react-native/commit/70cc27c901aeb447910e30ac3ceac85990d3c16d should fix it. Please let me know if it does not.
